cmd/image/qcow2ova/qcow2ova.go

```diff
@@ -37,9 +37,36 @@ Examples:
 
   # Converts the CentOS image from the local filesystem with OS password set
   pvsadm image qcow2ova --image-name centos-82 --image-dist centos --os-password s0meC0mplexPassword --image-url /root/CentOS-8-GenericCloud-8.2.2004-20200611.2.ppc64le.qcow2
+
+  # Customize the image preparation script for RHEL/CentOS distro, e.g: add additional yum repository or packages, change name servers etc. 
+  # Step 1 - Dump the default image preparation template
+  pvsadm image qcow2ova --prep-template-default > image-prep.template
+  # Step 2 - Make the necessary changes the above generated template file(bash shell script) - image-prep.template
```

Comment:
Make the necessary changes **to** the above generated template file(b

Reply:
done!